src/lib/plugin_apis/lvm.api

```diff
@@ -2044,4 +2045,34 @@ gboolean bd_lvm_devices_delete (const gchar *device, const gchar *devices_file,
  */
 gchar* bd_lvm_config_get (const gchar *section, const gchar *setting, const gchar *type, gboolean values_only, gboolean global_config, const BDExtraArg **extra, GError **error);
 
+/**
+ * bd_lvm_vgcfgbackup:
+ * @vg_name: name of the VG to backup
+ * @backup_file: (nullable): file to save the backup to or %NULL for using the default backup file
```

Comment:
I still get an impression that this is about backing up the whole VG data. I think it should be noted this is about _configuration_ of the VG.

Reply:
Ok, I've changed both the docstrings to emphasize the backup/restore is for configuration only. I've also added note from the `vgcfgbackup` which mentions this doesn't back up the LV data.